peer/brontide.go

```diff
@@ -88,8 +88,13 @@ type outgoingMsg struct {
 // the receiver of the request to report when the channel creation process has
 // completed.
 type newChannelMsg struct {
+	// channel is used when the pending channel becomes active.
 	channel *channeldb.OpenChannel
-	err     chan error
+
+	// channelID is used when there's a new pending channel.
+	channelID lnwire.ChannelID
```

Comment:
Isn't this channel ID already known for _all_ channels? Should it be nil/zero in certain cases?